### ISSUE TITLE
easy with data

### ISSUE THREAD

**Issue by ih4cku** [ih4cku/blog#65]
# learning with synthesis data
- [Pedestrian Detection by Synthesis](http://vishnu.boddeti.net/projects/detection-by-synthesis.html)
- [SynthText in the Wild Dataset](http://www.robots.ox.ac.uk/~vgg/data/scenetext/)
- [Synthetic Data for Scene Text Recognition](http://www.robots.ox.ac.uk/~vgg/research/text/)

**Comment by ih4cku** [comment]
# learning from web
- CVPR15 [Learning from Massive Noisy Labeled Data for Image Classification](https://github.com/Cysu/noisy_label)
- [Large-Scale Live Active Learning: Training Object Detectors with Crawled Data and Crowds](http://vision.cs.utexas.edu/projects/livelearning/)

**Comment by ih4cku** [comment]
In this \eqref{samp} equation.

$$e^i+1=0 \label{samp}$$

**Comment by ih4cku** [comment]
from https://github.com/facebook/fb.resnet.torch

> **Scale augmentation**: We use the [scale and aspect ratio augmentation](https://github.com/facebook/fb.resnet.torch/datasets/transforms.lua#L130) from [Going Deeper with Convolutions](http://arxiv.org/abs/1409.4842), instead of [scale augmentation](https://github.com/facebook/fb.resnet.torch/datasets/transforms.lua#L113) used in the ResNet paper. We find this gives a better validation error.
> 
> **Color augmentation**: We use the photometric distortions from [Andrew Howard](http://arxiv.org/abs/1312.5402) in addition to the [AlexNet](http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf)-style color augmentation used in the ResNet paper.
> 
> **Weight decay**: We apply weight decay to all weights and biases instead of just the weights of the convolution layers.
> 
> **Strided convolution**: When using the bottleneck architecture, we use stride 2 in the 3x3 convolution, instead of the first 1x1 convolution.

**Comment by ih4cku** [comment]
# augmentation
- http://arxiv.org/abs/1602.02660
